src/SUFST/inc/can_publisher.h

```diff
@@ -2,10 +2,21 @@
 #define CAN_PUBLISHER_H
 #include <stdint.h>
 #include <tx_api.h>
+#include "telemetry_protocol.h"
 
-extern TX_QUEUE queue;
-UINT queue_send_thread_create(TX_BYTE_POOL* stack_pool_ptr);
-void queue_send_thread_entry(ULONG input);
+#define TX_QUEUE_SIZE 10
```

Comment:
`TX_QUEUE_SIZE` could be confused for the name of a a ThreadX constant, it's unfortunate that `TX` is what ThreadX uses when it's so common to represent "transmit." I tend to have any constants for a module be prefixed by the module name. For example, here I would suggest `PUBLISHER_TX_QUEUE_SIZE`.